thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/template.hbs

```diff
@@ -1,31 +1,72 @@
 <div class="manage-alert-tune">
 
+  <div class="te-pill-selectors">
+    {{!-- Date range selector --}}
+    <div class="te-pill-selectors__label">Showing</div>
+    <ul class="te-alert-page-hzlist">
+      {{#each timeRangeOptions as |range|}}
+        <li class="te-alert-page-hzlist__item {{if range.isActive "te-alert-page-hzlist__item--active"}}" {{action "onRangeOptionClick" range}}>
+          {{range.name}}
+          {{#if (eq range.name "Custom")}}
+            : {{date-range-picker
+              class="te-pill-selectors__range-picker"
+              timePicker=false
+              timePicker24Hour=true
+              timePickerIncrement=timePickerIncrement
+              maxDate=today
+              start=viewRegionStart
+              end=viewRegionEnd
+              ranges=predefinedRanges
+              showCustomRangeLabel=false
+              format=uiDateFormat
+              serverFormat="YYYY-MM-DD HH:mm"
+              applyAction=(action "onRangeSelection")
+            }}
+          {{/if}}
+        </li>
+      {{/each}}
+    </ul>
+  </div>
+
   <div class="te-content-block">
     <fieldset class="te-form__section te-form__section--first te-form__section--slim row">
       <div class="col-xs-12">
         <legend class="te-form__section-title">Tune alert sensitivity</legend>
       </div>
       <ul class="te-form__list col-xs-12 col-sm-8">
         <li class="te-form__list-item">
-          {{#radio-button
-            value="current"
-            groupValue=selectedTuneType
-            changed=(action "onUpdateTuneType")
-            }}
-            <div class="te-form__list-label">Tune automatically with current settings</div>
+          {{#radio-button value="current" groupValue=selectedTuneType}}
+            <div class="te-form__list-label">Tune automatically with default settings</div>
           {{/radio-button}}
         </li>
         <li class="te-form__list-item">
-          {{#radio-button
-            value="custom"
-            groupValue=selectedTuneType
-            changed=(action "onUpdateTuneType")
-            }}
+          {{#radio-button value="custom" groupValue=selectedTuneType}}
             <div class="te-form__list-label">Tune with customized settings</div>
           {{/radio-button}}
          </li>
       </ul>
 
+      <div class="te-form__indent-block col-xs-8">
+        <span class="te-form__list-label">Anomaly Pattern</span>
```

Comment:
should we use `<label for="...."` here since we have an input with id ="alert-tune-pattern"

Reply:
good idea.